src/specialOffer/SpecialOffer.style.tsx

```diff
@@ -0,0 +1,47 @@
+import styled from 'styled-components'
+
+import { responsiveBreakpoints, space } from '../_utils/branding'
+import { normalizeHorizontally } from '../layout/layoutNormalizer'
+
+export const StyledSpecialOffer = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+`
+
+export const StyledSpecialOfferContent = styled.div`
+  flex: 1;
+  align-items: flex-start;
+  max-width: 50%;
+
+  @media (${responsiveBreakpoints.isMediaSmall}) {
+    max-width: 100%;
+    min-width: 100%;
+  }
+`
+
+export const StyledSpecialOfferFigure = styled.figure`
+  width: 70%;
```

Comment:
No need for a `max-width` anymore for desktop size screens because we switch to a 2-columns layout in that case.

Reply:
Is this "magic value" coming from somewhere else? Maybe we could factorize/centralize this for other kinds of desktop layout 🤷

Reply:
SpecialOffer layout is the very same layout that we can find at `SuccessModal`
On SuccessModal the  Media x Body proportion was 1/3 - 2/3. Which means 7-% is indeed a magic number.
Would be important to keep this on track when we are going to realign layouts.

Reply:
It's a magic value but for now I don't think it's used elsewhere. Let's wait for a new usage of this one before trying to factorize it.